src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -414,6 +425,22 @@ private void EnsureSignature()
         internal void MarkBackingFieldAsCalculated()
         {
             Interlocked.CompareExchange(ref _lazyBackingFieldSymbol, null, _lazyBackingFieldSymbolSentinel);
+            if (!IsOverride || _lazyBackingFieldSymbol is null)
+            {
+                return;
+            }
+
+            var fieldSymbol = (SynthesizedBackingFieldSymbol)_lazyBackingFieldSymbol;
+            if (fieldSymbol.IsCreatedForFieldKeyword &&
+                Interlocked.CompareExchange(ref _semiAutoPropertyHasGivenMustOverrideDiagnostic, new object(), null) == null)
```

Comment:
>Interlocked.CompareExchange(ref _semiAutoPropertyHasGivenMustOverrideDiagnostic, new object(), null) == null [](http://example.com/codeflow?start=16&length=108)

We don't need and do not want to keep track of the fact whether the error was reported already. Every time ```MethodCompiler``` calls this API, we should report the diagnostics because it is not going into any long lived cache.